### ISSUE TITLE
JSfiddle's in websites's "Examples" section don't work

### ISSUE THREAD

**Issue by git-toni** [recharts/recharts#317]
Is it just me? [link](http://recharts.org/examples)

**Comment by konstantin24121** [comment]
@git-toni Not only for you. All examples down

**Comment by git-toni** [comment]
Yup, `'t' is not a function' somewhere in the bundled `min.js` version.

**Comment by git-toni** [comment]
```javascript
//Within Recharts.js
...
	function baseOrderBy(collection, iteratees, orders) {
	  var index = -1;
	  iteratees = arrayMap(iteratees.length ? iteratees : [identity], baseUnary(baseIteratee));

	  var result = baseMap(collection, function(value, key, collection) {
	    var criteria = arrayMap(iteratees, function(iteratee) {
	      return iteratee(value);   //<---- Error here
	    });
	    return { 'criteria': criteria, 'index': ++index, 'value': value };
	  });

	  return baseSortBy(result, function(object, other) {
	    return compareMultiple(object, other, orders);
	  });
	}
...
```

**Comment by xile611** [comment]
Fixed in version 0.16.1.
